app/models/follow_up.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+require 'rest-client'
+
+class FollowUp
+  include ActiveModel::Validations
+
+  attr_accessor :email, :name, :language, :destination
+
+  validates :email, presence: true, email_format: { message: 'Invalid email address' }
+  validates :language, presence: true
+  validates :destination, presence: true
+
+  def initialize(email, language_id, destination_id, name = nil)
+    self.email = email
+    self.language = Language.find(language_id)
```

Comment:
Just making sure you know that the `find` method will throw an exception if for some reason it can't find a record with that id. From what I can tell, the id is coming in from an API call, which means it could be a bad id. You might want to rescue the possible exception in the controller and respond with a friendly error message. Otherwise they'll just get a 500 and think that the system failed when really it was bad input.